src/tools/dashboard/src/util.rs

```diff
@@ -185,7 +185,16 @@ pub fn add_codegen_job(litani: &mut Litani, test_props: &TestProps) {
     let mut rmc_rustc = Command::new("rmc-rustc");
     rmc_rustc
         .args(&test_props.rustc_args)
-        .args(["-Z", "codegen-backend=gotoc", "--cfg=rmc", "--out-dir", "build/tmp"])
+        .args([
+            "-Z",
+            "codegen-backend=gotoc",
+            "--cfg=rmc",
+            "--out-dir",
+            "build/tmp",
+            "-Z",
+            "human_readable_cgu_names",
+            "-crate-type=lib",
```

Comment:
Should be two dashes.

Reply:
Good catch... I wonder when this gets run. I should check with @adpaco-aws to make sure we have test coverage in our regression.

Reply:
We have the dashboard set to only run after merges. I think there are concerns right now about prolonging CI times.

Reply:
Anyway, I fixed it and I ran the dashboard script locally. The output is a bit hard to parse, but it seemed to work.